APIs/RegistrationAPI.raml

```diff
@@ -82,13 +82,16 @@ documentation:
             example: /x-nmos/registration/{version}/resource/nodes/3b8be755-08ff-452b-b217-c9151eb21193/
         description: The expected response for a create operation performed for a previously unregistered resource
       400:
-        description: Returned when the POST request is incorrectly formatted or missing mandatory attributes
+        description: Returned when the POST request is incorrectly formatted, missing mandatory attributes or breaches another condition which a Node is unlikely to be able to automatically correct.
         body:
           type: ErrorSchema
       409:
-        description: Returned when the resource already exists in the registry at a different API version
+        description: Returned when the resource already exists in the registry at a different API version. The correct API version is identified via the Location header.
         body:
           type: ErrorSchema
+        headers:
+          Location:
+            example: /x-nmos/registration/{version}/resource/nodes/3b8be755-08ff-452b-b217-c9151eb21193/
```

Comment:
From [URLs: Approach to Trailing Slashes](https://github.com/AMWA-TV/nmos-discovery-registration/blob/v1.3-dev/docs/2.0.%20APIs.md#urls-approach-to-trailing-slashes):
> For consistency and in order to adhere to how these APIs are specified in RAML, the 'primary' path for every resource has the trailing slash omitted.

I realise that applies to the other pre-existing Location examples too.

Reply:
Yes... Unfortunately the spec as a whole has a habit of presenting URLs with a slash in examples, but requiring that actual interactions with them remove the slash. I was conflicted as to whether to 'correct' this in these new cases or not.

Reply:
OK ☹️
Add a low-priority issue to fix all such examples?

Reply:
Added to https://github.com/AMWA-TV/nmos-discovery-registration/issues/87